src/Message/ApplePayAuthorizeResponse.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Omnipay\Vindicia\Message;
+
+class ApplePayAuthorizeResponse extends Response
+{
+    /**
+     * Was the response successful?
+     *
+     * @return boolean
+     */
+    public function isSuccessful()
+    {
+        /**
+         * @var boolean
+         */
+        return ($this->data['statusCode'] == 200);
+    }
+
+    /**
+     * The entire Apple Pay payment session object to be used to validate the merchant.
+     * This should be passed to the front end.
+     *
+     * @return static
```

Comment:
i don't think this return type is correct. static would be a Response object, but it looks like it returns a string